docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+
+services:
+  grafana:
+    image: grafana/grafana-oss:9.1.0-ubuntu
+    volumes:
+      - .:/var/lib/grafana/plugins/grafana-checkmk-datasource
+    environment:
+      GF_ANALYTICS_REPORTING_ENABLED: 'false'
+      GF_ANALYTICS_CHECK_FOR_UPDATES: 'false'
+      GF_SECURITY_ADMIN_USER: admin
+      GF_SECURITY_ADMIN_PASSWORD: password
+      GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS: tribe-29-checkmk-datasource
+    ports:
+      - '3000:3000'
+    depends_on:
+      - checkmk
+  checkmk:
+    image: checkmk/check-mk-free:2.1.0p9
```

Comment:
should we pin the versions, or always use the latest version in order to run into problems?
if we use `:latest` for the checkmk tag, perhaps we should also create a weekday job to run the e2e action in order to early detect problems.

Reply:
We should use `2.1-latest` and grafana `9-latest` or what ever there is.

Reply:
Sadly there is only 'main' and 'latest' for Grafana besides specific versions.